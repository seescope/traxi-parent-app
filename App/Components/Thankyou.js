import React from 'react';
import { Dimensions, Platform, Text, Image } from 'react-native';
import { WHITE, TRANSPARENT } from '../Constants/Colours';
import Spacing from '../Components/Spacing';

const { width } = Dimensions.get('window');

const containerStyle = {
  paddingTop: 32,
  paddingHorizontal: 32,
  flex: 1,
  width,
};

const bodyStyle = {
  paddingBottom: 16,
  backgroundColor: TRANSPARENT,
  color: WHITE,
  fontSize: Platform.select({
    ios: 16,
    android: 14,
  }),
  textAlign: 'left',
  fontFamily: 'Raleway-Regular',
};

const headerStyle = {
  backgroundColor: TRANSPARENT,
  color: WHITE,
  fontSize: Platform.select({
    ios: 26,
    android: 24,
  }),
  textAlign: 'center',
  fontFamily: 'Raleway-ExtraBold',
};

export default () =>
  <Image
    style={containerStyle}
    source={require('../Images/thankyou-background.png')}
  >
    <Text style={headerStyle}>Thanks!</Text>

    <Spacing height={36} />

    <Text style={bodyStyle}>
      We'll send you a reminder tomorrow.
    </Text>

    <Text style={bodyStyle}>
      Remember, you can come back to the app and start monitoring your kids at any time.
    </Text>
  </Image>;
