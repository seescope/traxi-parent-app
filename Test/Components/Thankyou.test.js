import React from 'react';
import renderer from 'react-test-renderer';

import Thankyou from '../../App/Components/Thankyou';

it('renders the <Thankyou> component', () => {
  const tree = renderer.create(<Thankyou />).toJSON();

  expect(tree).toMatchSnapshot();
});
