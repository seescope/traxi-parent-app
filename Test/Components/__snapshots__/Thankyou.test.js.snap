exports[`test renders the <Thankyou> component 1`] = `
<Image
  source={1}
  style={
    Object {
      "flex": 1,
      "paddingHorizontal": 32,
      "paddingTop": 32,
      "width": 750,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "#FFF",
        "fontFamily": "Raleway-ExtraBold",
        "fontSize": 26,
        "textAlign": "center",
      }
    }>
    Thanks!
  </Text>
  <View
    style={
      Object {
        "height": 36,
      }
    } />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "#FFF",
        "fontFamily": "Raleway-Regular",
        "fontSize": 16,
        "paddingBottom": 16,
        "textAlign": "left",
      }
    }>
    We\'ll send you a reminder tomorrow.
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "#FFF",
        "fontFamily": "Raleway-Regular",
        "fontSize": 16,
        "paddingBottom": 16,
        "textAlign": "left",
      }
    }>
    Remember, you can come back to the app and start monitoring your kids at any time.
  </Text>
</Image>
`;
